docs/src/tutorials/dc_motor_pi.md

```diff
@@ -0,0 +1,93 @@
+# DC Motor with PI-controller
+In this example a PI-controller is setup for speed control of a DC-motor. First the needed packages 
+are imported and the parameters of the model defined.
+
+```@example dc_motor_pi
+using ModelingToolkit
+using ModelingToolkitStandardLibrary.Electrical
+using ModelingToolkitStandardLibrary.Mechanical.Rotational
+using ModelingToolkitStandardLibrary.Blocks
+using OrdinaryDiffEq
+using Plots
+
+@parameters t
+
+R = 0.5 # [Ohm]
+L = 4.5e-3 # [H]
+k = 0.5 # [N.m/A]
+J = 0.02 # [kg.m²]
+f = 0.01 # [N.m.s/rad]
+tau_L_step = -3 # [N.m]
+nothing # hide
+```
+
+The actual model can now be composed.
+
+```@example dc_motor_pi
+@named ground = Ground()
+@named source = Voltage()
+@named ref = Blocks.Step(height = 1, start_time = 0)
+@named pi_controller = Blocks.LimPI(k = 1.1, T = 0.035, u_max = 0.6, Ta = 0.035)
+@named feedback = Blocks.Feedback()
+@named R1 = Resistor(R = R)
+@named L1 = Inductor(L = L)
+@named emf = EMF(k = k)
+@named fixed = Fixed()
+@named load = Torque(use_support = false)
+@named load_step = Blocks.Step(height = tau_L_step, start_time = 3)
+@named inertia = Inertia(J = J)
+@named friction = Damper(d = f)
+@named speed_sensor = SpeedSensor()
+
+connections = [connect(fixed.flange, emf.support, friction.flange_b)
+               connect(emf.flange, friction.flange_a, inertia.flange_a)
+               connect(inertia.flange_b, load.flange)
+               connect(inertia.flange_b, speed_sensor.flange_a)
+               connect(load_step.output, load.tau)
+               connect(ref.output, feedback.input1)
+               connect(speed_sensor.w_rel, feedback.input2)
+               connect(feedback.output, pi_controller.err_input)
+               connect(pi_controller.ctr_output, source.input)
+               connect(source.p, R1.p)
+               connect(R1.n, L1.p)
+               connect(L1.n, emf.p)
+               connect(emf.n, source.n, ground.g)]
+
+@named model = ODESystem(connections, t,
+                         systems = [
+                             ground,
+                             ref,
+                             pi_controller,
+                             feedback,
+                             source,
+                             R1,
+                             L1,
+                             emf,
+                             fixed,
+                             load,
+                             load_step,
+                             inertia,
+                             friction,
+                             speed_sensor,
+                         ])
+nothing # hide
+```
+
+Now the model can be simulated. Typical rotational mechanical systems are described via `DAE` 
+(differential algebraic equations), however in this case ModelingToolkit can simplify the model enough
+so that it can be represented as a system of `ODEs` (ordinary differential equations).
+
+```@example dc_motor_pi
+sys = structural_simplify(model)
```

Comment:
Due to all `Block.RealInputs` being marked as `input=true` even the simplified system has a whopping 19 states:
```julia 
julia> states(sys)
19-element Vector{Term{Real, Base.ImmutableDict{DataType, Any}}}:
 pi_controller₊int₊x(t)
 L1₊i(t)
 inertia₊phi(t)
 inertia₊w(t)
 pi_controller₊gainPI₊input₊u(t)
 pi_controller₊addPI₊input1₊u(t)
 pi_controller₊addPI₊input2₊u(t)
 pi_controller₊int₊input₊u(t)
 pi_controller₊addTrack₊input1₊u(t)
 pi_controller₊addTrack₊input2₊u(t)
 pi_controller₊limiter₊input₊u(t)
 pi_controller₊addSat₊input1₊u(t)
 pi_controller₊addSat₊input2₊u(t)
 pi_controller₊gainTrack₊input₊u(t)
 feedback₊input2₊u(t)
 feedback₊input1₊u(t)
 source₊V₊u(t)
 load₊tau₊u(t)
 pi_controller₊err_input₊u(t)
```